mitosheet/src/mito/types.tsx

```diff
@@ -997,7 +997,8 @@ export interface BaseAction<Type, StaticType> {
     type: Type;
     staticType: StaticType;
     // The short title for the action. Should be title case, as you want to display it.
-    shortTitle: string
+    // If undefined, the toolbar will not display a title in the toolbar. 
```

Comment:
Can we rename shortTitle to just `toolbarTitle`? I think this removes the need for a comment actually.

Reply:
Updated the name! Leaving the comment in because I think it's expressing something slightly different